qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -1572,3 +1592,8 @@ def num_basis_gates(self, unitary):
 
 
 two_qubit_cnot_decompose = _LazyTwoQubitCXDecomposer()
+"""
+This is an instance of :class:`.TwoQubitBasisDecomposer` that always uses
+``cx`` as the KAK gate for the basis decomposition. You can use this function
+as a quick access to ``cx``-based 2-qubit decompositions.
+"""
```

Comment:
If I'm mistaken the return type is not accurate, maybe we can specify it manually too:

```suggestion
as a quick access to ``cx``-based 2-qubit decompositions.

Returns:
    :class:`.TwoQubitBasisDecomposer`: `cx``-based 2-qubit decomposer instance.
"""
```